scico/linop/radon_svmbir.py

```diff
@@ -57,8 +57,8 @@ def __init__(
             angles: Array of projection angles in radians, should be
                 increasing.
             num_channels: Number of pixels in the sinogram.
-            center_offset: Position of the detector center relative to the center-of-rotation,
-                in units of pixels
+            center_offset: Position of the detector center relative to
+                the center-of-rotation, in units of pixels
```

Comment:
Hyphenation necessary in center-of-rotation?

Reply:
No. Fixed.